templates/tfengine/components/cicd/main.tf

```diff
@@ -117,12 +115,11 @@ resource "google_project_iam_member" "cloudbuild_builds_viewers" {
     google_project_service.services,
   ]
 }
-{{- end}}
-
-{{- if has . "build_editors"}}
 
 # IAM permissions to allow approvers to edit/create the cloud build jobs.
 resource "google_project_iam_member" "cloudbuild_builds_editors" {
+  count    = length(var.build_editors) > 0 ? 1 : 0
```

Comment:
This is not needed i believe. This resource already has a for_each block. Same thing for the build viewers.

Reply:
You're right, I'm also questioning about what will happen if both are specified 🧐

Fixed anyway